chart/kubeapps/templates/apprepository-rbac.yaml

```diff
@@ -142,4 +142,19 @@ rules:
       - secrets
     verbs:
       - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: "kubeapps:{{ .Release.Namespace }}:apprepositories-webhook"
```

Comment:
I'd call this `kubeapps:{{ .Release.Namespace }}:apprepositories-refresh` to follow the same naming than the others.

Reply:
Good idea. Since it involves changing the docs, I will also send a PR for that purpose.